DOCS-5584 customization (#98)
mulesoft/docs-api-community-manager: modules/ROOT/pages/customize.adoc

> = API Community Look and Feel Customization

Reviewer: ```suggestion
= API Community Look-and-Feel Customization
```
Hyphenated as a compound adject; as a noun, not.

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> The https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables

Reviewer: ```suggestion
https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] enables
```
Never precede a proper noun with an article, unless the proper noun is followed by a common noun object.
GLOBALLY standardize references to Community Builder; avoid just Builder.

you to create a community with your desired brand identity, look and feel, and
site structure by providing control over the community's components, themes,

> pages, and its general, theme, and SEO settings.

Reviewer: ```suggestion
and pages, and its general theme and SEO settings.
```

> In the API Community Manager (ACM) Control Panel, the *Community Builder* shows

Reviewer: ```suggestion
In the API Community Manager (ACM) control panel, the *Community Builder* [panel? screen? page?] shows
```

your site for editing. The builder is where you edit each page in your
community, create new pages, and add and edit the components in each page.

> The *Community Builder* interface contains the elements described here.

Reviewer: ```suggestion
The Community Builder interface contains the elements described here.
```

Reviewer: However, I suggest narratively listing the elements rather than using "the elements described here."
This sets the expectation for the following subsections.

Author: Narratively listed elements.

== Pages List

The *Pages* list shows the pages included with ACM and the pages you have added,
and enables you to select and edit each page. The included page *Home* is
selected by default.

> The *Pages* list enables you to create new pages with the *New Page* option.

Reviewer: ```suggestion
The *Pages* list enables you to create new pages by using the *New Page* option.
```

> A new page can be an *Object Page* or *Standard Page*. An *Object Page* is used

Reviewer: ```suggestion
A new page can be an object page or a standard page. An object page is used
```
In this context, these are types of pages and not references to their appearance in the UI.

only with Salesforce objects, so a custom community page starts as a

> *Standard Page*. A standard page can start from one of the preconfigured pages

Reviewer: ```suggestion
standard page. A standard page can start from one of the preconfigured pages
```

> or as a *New Blank Page*.

Reviewer: ```suggestion
or as a new blank page.
```

> A new blank page is made by selecting the layout and then entering the new

Reviewer: Is all the passive voice intentional here? Recommend recasting as things that your audience can do, not as things that can be acted upon: is made, is used, and so on.

Author: The previous version was an awkward mix between a task topic and a concept topic, so I rewrote it as a concept topic, but it did introduce lots of passive voice. I've rewritten to avoid passive voice.

> page's *Name* and *URL*. The URL is used to make links to the new page,

Reviewer: ```suggestion
page's name and URL. The URL is used to make links to the new page,
```

including links in menus.

The new page is displayed with placeholders for content based on the layout.

> A new page is filled in by adding components from the *Components* menu, such

Reviewer: passive

as a *Headline* component in the page header, or an ACM Lightning component

> such as an *API Version Carousel*.

Reviewer: Do not boldface component names.

The *Pages* list shows the editable configuration properties for each page,
which are described in https://help.salesforce.com/articleView?id=community_builder_manage_pages_overview.htm&type=5[Manage Your Community’s Pages and Their Properties in Community Builder, role=external, window=blank].

== Components

> The *Components* lightning bolt icon displays the component palette, which

Reviewer: ```suggestion
Clicking the *Components* lightning bolt icon displays the component palette, which
```

Reviewer: GLOBALLY unbold the component names; I will stop marking from here.

Author: Unbolded component names.

includes Salesforce Community Cloud components, such as *Headline* or
*Featured Topics*, and ACM-specific components, such as *API Carousel* or

> *API Console*. Each component can be dragged from the component palette to the

Reviewer: passive

current page. Selecting the component on the page shows the component's
configuration options.

Detailed component information is available from the page xref:acm-lightning-components.adoc[Description of ACM Lightning Components] and
the Salesforce Community Cloud https://help.salesforce.com/articleView?id=rss_component_reference_table.htm&type=5[Community Components, role=external, window=blank] documentation.

> Components typically used in *Home* pages include:

Reviewer: ```suggestion
Components typically used in home pages include:
```

* https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
* https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
* *API Carousel*
+
Displays APIs and API versions that are available to the community from Anypoint Exchange.
+
* *API Details Button*
+
Provides details about an API.
+
* https://help.salesforce.com/articleView?id=community_builder_rich_text_component.htm&type=5[Rich text editor, role=external, window=blank]
* https://help.salesforce.com/articleView?id=rss_html_editor.htm&type=5[HTML editor, role=external, window=blank]
* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]

The *Navigation Menu* component provides the menu for navigating the site, with
settings for *Max Items to Display (2-15)* and *Menu Item Alignment*, and a
*Navigation Menu* button to edit the links on the navigation menu. The *Home*

> link is always present and is always the first item. Other links can be added,

Reviewer: passive

> removed, and reordered. Each link other than *Home* can be customized with these

Reviewer: passive
GLOBALLY review for passive voice, and determine whether you intend to use it or not. See style ref.

options:

* *Name*
+
Sets the link name.
+
* *Type*
+
Sets the link type, such as *Community Page* or *External URL*.
+
* *Page*
+
Specifies community page links from a list of all community pages.
+
* *Publicly available*
+
Sets whether the link is available to all page viewers or only logged in
community members.

> Components typically used in *API details* pages include:

Reviewer: ```suggestion
Components typically used in API details pages include:
```
...and do not boldface any of the following component names.

* *API Header*
+
Specifies the API name and description.
+
* *Tabs*
+
Contains these components:
+
** *API Documentation Viewer*
+
Displays API Exchange information.
+
** *API Console*
+
Renders information from the API RAML specification.
+
** *Feed*
+
Displays comments, discussions, and questions.
+
* *API Access Requestor*
+
Registers a client application and requests access to the API.
+

> * *Learn More Button*

Reviewer: Is this a component? Button is part of the name?

Author: Correct, the component name is "Learn More Button".

+
Provides additional information about the API.

Components typically used in *My Application* pages include:

* *Application Listing*
+
Lists all client applications that the member has registered and can access.

Items on a page can be dragged into any order.

== Theme

> The *Theme* paintbrush icon offers these general branding parameters:

Reviewer: ```suggestion
Clicking the *Theme* paintbrush icon offers these general branding parameters:
```

* *Colors*
+

> Sets the site's *General* color scheme, including paragraph, heading, and button text.

Reviewer: ```suggestion
Sets the site's general color scheme, including paragraph, heading, and button text.
```

+
* *Images*
+

> Specifies the site's *Company Logo* and the login page's *Background Image*.

Reviewer: ```suggestion
Specifies the site's company logo and the login page's background Image.
```

+
* *Fonts*
+

> Specifies the site's fonts, which include a *Primary Font* and *Header Fonts*,

Reviewer: unbold and lowercase all these terms

Author: Done.

and specifies the site's *Text Case*.
+
* https://help.salesforce.com/articleView?id=community_builder_theme_adjust.htm&type=5[Theme Settings, role=external, window=blank]

Community styling can be adjusted by editing the CSS rules. Next to the *Theme*

> title is the *Theme* list which offers the option *Edit CSS*.

Reviewer: ```suggestion
title is the *Theme* list, which offers the option *Edit CSS*.
```

Refer to https://help.salesforce.com/articleView?id=community_builder_theme_customize.htm&type=5[Customize Your Community Theme, role=external, window=blank].

== Page Structure

> The *Page Structure* icon, which resembles a bulleted list, shows the components

Reviewer: ```suggestion
Clicking the *Page Structure* icon, which resembles a bulleted list, shows the components
```

in the current page and enables you to select and remove them. Selecting a
component displays its configuration menu.

== Settings

> The *Settings* gear icon offers these parameters:

Reviewer: ```suggestion
Clicking the *Settings* gear icon offers these parameters:
```
